polymetis/tests/scripts/6_gravity_comp.py

```diff
@@ -0,0 +1,14 @@
+import torch
```

Comment:
Rename as `6_gravity_comp_hw.py` to avoid adding this to the integration tests (I don't think it'll terminate)

Reply:
Done